src/background/stores/assessment-card-selection-store.ts

```diff
@@ -64,6 +76,28 @@ export class AssessmentCardSelectionStore extends PersistentStore<AssessmentCard
         return defaultValue;
     }
 
+    private onAssessmentStoreChanged = async (
+        payload: AssessmentStoreChangedPayload,
+    ): Promise<void> => {
+        if (!payload || !payload.assessmentStoreData) {
```

Comment:
nit: check that payload.assessmentStoreData.assessment is non-null and has more than 0 entries here too

Reply:
I was thinking about this too but wasn't completely sure if we wanted to clear the card selection store state if the assessment store state was cleared (on starting over assessment), but I'll double check how the assessment store state looks after such an event.

Reply:
It appears that the assessment store state should never be empty or null, so I'll add the check for payload.assessmentStoreData.assessments as well. Thanks!